### ISSUE TITLE
Publish GitLens in Open VSX

### ISSUE THREAD

**Issue by tdemin** [gitkraken/vscode-gitlens#1102]
There's an alternate [open source](https://open-vsx.org/about) extension marketplace for VSCode, [Open VSX](https://open-vsx.org).

As Microsoft's extensions marketplace cannot be used by Code-OSS and VSCodium users [for legal reasons](https://github.com/microsoft/vscode/issues/31168#issuecomment-498803448), it's probably a good idea to make GitLens available to them there.

The process of publishing an extension at Open VSX is outlined [here](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions#how-to-publish-an-extension).

**Comment by orblazer** [comment]
This is published on vsx but @eamodio have not make auto update on this.

**Comment by tdemin** [comment]
@orblazer it's not published by @eamodio, see this text on [the addon page](https://open-vsx.org/extension/eamodio/gitlens):

> The “GitLens — Git supercharged” extension was published by open-vsx. This user account is not related to the namespace eamodio of this extension

The version there is also severely outdated and is currently at 10.2.1.

**Comment by orblazer** [comment]
Yes but she have claim the namespace : https://github.com/eclipse/open-vsx.org/issues/9

**Comment by Rob4226** [comment]
+1 Please add to Open VSX

**Comment by GabrielPPavan** [comment]
Any news on this? Ive gotten early access to Google IDX and its lacking gitlens...
I anticipate that when IDX launches for good, the demand for this will increase drastically...

**Comment by d13** [comment]
Resolved with ddd7ae94d3e4f9b430eff3527bbe59be36fda619
